style.css

```diff
@@ -522,10 +514,24 @@ q:after {
 	text-rendering: auto;
 }
 
+/* Typography for Thai Font */
+
+html[lang="th"] h1, h2, h3, h4, h5, h6 {
```

Comment:
@josephfusco You are right! All selectors should apply for Thai Font only. Can I fix and commit again?